vector/src/main/java/im/vector/app/features/home/room/detail/composer/MessageComposerView.kt

```diff
@@ -40,7 +46,7 @@ class MessageComposerView @JvmOverloads constructor(
         context: Context,
         attrs: AttributeSet? = null,
         defStyleAttr: Int = 0
-) : ConstraintLayout(context, attrs, defStyleAttr) {
+) : ConstraintLayout(context, attrs, defStyleAttr), MessageComposer {
```

Comment:
For clarity, I would rename this class from `MessageComposerView` to `MessageComposerLayout` (or `MessageComposerLegacyLayout`) and your new interface could take the name `MessageComposerView`. WDYT?